data/initrd/initrd.file_list

```diff
@@ -35,6 +35,7 @@ TEMPLATE:
 
 AUTODEPS:
 
+binutils: ignore
```

Comment:
Um, there are no details, I think it would be nice to add a comment, e.g. why we need to ignore the dependency and possibly why it is safe for us. As I can see we do not mention this for the other packages, but IHMO we should start with it.

It will be hard to review these exceptions in the future if do not know why they have been added. (Digging in the Git history is not a nice solution....)

Reply:
It's explained in the commit message. Note sure the code gets better adding an explanation to every line.